pandas/io/orc.py

```diff
@@ -49,6 +49,8 @@ def read_orc(
     if distutils.version.LooseVersion(pyarrow.__version__) < "0.13.0":
         raise ImportError("pyarrow must be >= 0.13.0 for read_orc")
 
+    import pyarrow.orc
```

Comment:
pyarrrow should only be imported once, so should remove the ```import pyarrow``` statement a couple lines up.

Reply:
Line-49 will check the Pyarrow's version, it depends on line-47(`import PyArrow`).

Reply:
Looking again, it's better to use https://github.com/pandas-dev/pandas/blob/04f9a4b10df91e6f4838dd8e1e41bd97141b6332/pandas/compat/_optional.py#L63
so would be
```
from pandas.compat._optional import import_optional_dependency
orc = import_optional_dependency("pyarrow.orc")
...
orc_file = orc.ORCFile(handles.handle)
```

Reply:
I already tested it and it looks good. But it will remind users to install `pyarrow.orc` instead of `pyarrow` since import_optional_dependency cannot receive customized error messages. Is that OK?

Log:
`ImportError: Missing optional dependency 'pyarrow.orc'.  Use pip or conda to install pyarrow.orc.`